### DISCUSSION TITLE
where are you get your icons from?

### DISCUSSION THREAD

**Discussion by mrsavari** [alibaba/designable#48] [Q&A]
i intend to add some features , where are you get your icons from?

**Comment by janryWang** [comment]
https://github.com/alibaba/designable/tree/main/packages/react/src/icons

**Reply by mrsavari** [reply]
this is the path of the used icons not the original icons source pack?

**Reply by janryWang** [reply]
all this icons is svg file by jsx format

**Reply by mrsavari** [reply]
is they are, but where the svg files are from i what to add new icons with the same design system, for example ant-design and material icons all svgs but with different design system.

**Reply by janryWang** [reply]
Haha, this icons designed by myself

**Reply by mrsavari** [reply]
great work, well done, respect 😎

**Comment by janryWang** [comment]
Welcome PR
